.vib/logstash/goss/vars.yaml

```diff
@@ -0,0 +1,27 @@
+binaries:
+  - java
+  - logstash
+  - logstash-plugin
+  - yq
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/logstash
+      - /bitnami/logstash/config
+      - /bitnami/logstash/data
+      - /bitnami/logstash/pipeline
+      - /opt/bitnami/logstash
+      - /opt/bitnami/logstash/config
+      - /opt/bitnami/logstash/pipeline
+      - /opt/bitnami/logstash/logs
+      - /opt/bitnami/logstash/plugins
+      - /opt/bitnami/logstash/tmp
+      - /docker-entrypoint-initdb.d
```

Comment:
AFAIK, we are not modifying these directories nor their permissions, are these checks necessary?

Reply:
You are right. I think I got confused with the files created with the VM, but the `logs`, `tmp` and `plugins` folders were not needed so I've removed them.